pkg/monitortests/network/disruptionpodnetwork/monitortest.go

```diff
@@ -126,12 +128,20 @@ func (pna *podNetworkAvalibility) StartCollection(ctx context.Context, adminREST
 		return err
 	}
 
+	irEnabled, err := imageRegistryEnabled(adminRESTConfig)
```

Comment:
`imageRegistryEnabled`.  don't be stingy.

Reply:
lol I was half in half out on the ʕ◔ϖ◔ʔ mentality, updating